src/main/java/com/partymusicapp/Controllers/PhotoControllerImpl.java

```diff
@@ -0,0 +1,63 @@
+package com.partymusicapp.Controllers;
+
+import com.partymusicapp.models.Photo;
+import com.partymusicapp.service.PhotoService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.IOException;
+import java.util.List;
+
+@RestController
+@RequestMapping("/pma/api/v1/upload")
+@RequiredArgsConstructor
+public class PhotoControllerImpl implements PhotoController {
+
+    private final PhotoService photoService;
+
```

Comment:
Nu trebuie sa ai implemntarea in controller la business logic, ci in service. Metoda de la service trebuie sa iti returneze un ResponseEntity<ceva> pe care metoda din controller o returneaza mai departe